backend/src/main/scala/cromwell/backend/wdl/WriteFunctions.scala

```diff
@@ -27,8 +32,15 @@ trait WriteFunctions extends IoFunctionSet with AsyncIoFunctions {
     }
   }
 
+  private val relativeToLocal = {
+    val initial = System.getProperty("user.dir")
+    if (initial.endsWith("/")) initial else initial + "/"
```

Comment:
I feel like I've seen 37 different versions of this line in various PRs recently